index.html

```diff
@@ -26,20 +34,24 @@
     
 
     <section class="presentation">
-        <h1 class="text-1">
-            Hey There.
-            <div>I’m Edahi</div>    <!-- we need this div to insert a return line -->
-        </h1>
+
+        <div class="wrapper-extra">
+            <h1 class="text-1">
+                Hey There.
+                <div>I’m Edahi</div>    <!-- we need this div to insert a return line -->
+            </h1>
+            
+            <h2 class="text-2">
+                I’am a Software Developer
+            </h2>
+            <p class="text-3">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.            
+            </p>            
```

Comment:
As your first reviewer pointed out in the video, please take care of this overlapping text and image in order to provide a good user experience to page visitors.


**Your design**

<img width="659" alt="cryyyyy" src="https://user-images.githubusercontent.com/84930355/206859325-7ce89c23-7b1d-4ba7-b6c0-87e00dc1d497.png">


TIP: on different breakpoints, reduce the width of the image and adjust the font size of the text.